### PR TITLE
Add Sub Resolution Option When Closing An Insight

### PR DESCRIPTION
**Pull request by yleiferman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added an option to set up a sub resolution in the CLI when closing an insight.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by aaron1535** [comment]
Hi @yleiferman,
Thank you for your contribution.
So we can continue your PR
Please make sure to register your contribution by filling the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form.

**Comment by yleiferman** [comment]
Hi @aaron1535 , please let me know if there's anything else that you would like me to do. Thank you for your help.

**Review by aaron1535 — CHANGES_REQUESTED** [review]
Hi @yleiferman
good work
Check my comments please so we can move forward with the merger

File: package-lock.json [hunk 1]
```diff
@@ -1,1057 +1,1500 @@
...
-    "dependencies": {
-      "@babel/code-frame": {
-        "version": "7.23.5",
-        "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.23.5.tgz",
-        "integrity": "sha512-CgH3s1a96LipHCmSUmYFPwY7MNx8C3avkq7i4Wl3cfa662ldtUe4VM1TPXX70pfmrlWTb6jLqTYrZyT2ZTJBgA==",
-        "requires": {
-          "@babel/highlight": "^7.23.4",
-          "chalk": "^2.4.2"
-        }
-      },
```
[the remark this answers is not in the document]
Comment (yleiferman) [reply package-lock.json]: Done @aaron1535 . I've removed it. Thank you for the feedback.

**Review by aaron1535 — APPROVED** [review]

**Pull request merged**